examples/node/MarketDataSubscription.js

```diff
@@ -32,7 +32,7 @@ socket.on('data', function (data) {
     console.log('Data Count: ' + counter++);
     console.log(data);  
     if (counter === NUM) {
-        socket.emit('unsubscribe');
+        socket.emit('unsubscribe', { correlationIds: [0, 1] });
```

Comment:
This should not be needed here. The intention is that if user send `unsubscribe` event without `correlationIds`, the server will unsubscribe all active subscriptions.